CMakeLists.txt

```diff
@@ -94,6 +94,11 @@ else()
   list(APPEND _glvis_compile_defs "_USE_MATH_DEFINES")
 endif()
 
+find_program(XXD_FOUND xxd)
+if(NOT XXD_FOUND)
+    message(FATAL_ERROR "xxd not found!")
+endif()
```

Comment:
```suggestion
if(NOT WIN32)
  find_program(XXD_FOUND xxd)
  if(NOT XXD_FOUND)
    message(FATAL_ERROR "Required tool not found: xxd.")
  endif()
endif()
```

Reply:
Nice catch. Totally forgot about the windows support.